LICENSE

```diff
@@ -0,0 +1,203 @@
+Copyright 2017-2020 Determined.ai, Inc.  All rights reserved.
```

Comment:
This is probably fine for now? At some point (soon) should be HPE though.

Reply:
We might as well get the date range correct though :)

Reply:
dates fixed.